components/meter/test/meter-linear.visual-diff.js

```diff
@@ -21,9 +21,10 @@ describe('d2l-meter-linear', function() {
 		{ title: 'no-progress', fixture: '#no-progress'},
 		{ title: 'has-progress', fixture: '#has-progress'},
 		{ title: 'completed', fixture: '#completed'},
-		{ title: 'max-zero', fixture: '#max-zero'},
+		{ title: 'max-zero-value-zero', fixture: '#max-zero-value-zero'},
```

Comment:
Since this test was renamed, you'll want to rename the diff file that's checked in at `test/screenshots/ci/meter-linear/d2l-meter-linear-max-zero.png`.

Reply:
It must be too early in the morning since you told me to do this :D